### DISCUSSION TITLE
2.6.10 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#2812] [讨论]
· 更新 Mod 数据库
· 帮助库小修改（by @WForst-Breeze、@WTP016-CN、@wuyuncheng-26）
· 修复了以下 3 个 Bug
　　在 Mod/整合包 详情加载完之前返回导致出错（#2761，@212112111zwl）
　　在 MC 更新提示中点击下载导致出错（#2806，@WForst-Breeze）
　　罕见情况下启用/禁用 Mod 导致出错（#2807，@SuoHouDaiShi）

已知 Bug
· 版本号末尾多了几个 0，实际版本号是 2.6.10，这只是一个显示问题，没有实际影响

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.10'>2.6.10</a>.</em>

**Comment by SuoHouDaiShi** [comment]
<del>罕见带师</del>
为什么我每次交的BUG都是罕见情况？？

**Reply by MoYuan-CN** [reply]
因为你的 Bug 真的 很！罕！见！（迫真

**Reply by SuoHouDaiShi** [reply]
<del>十分甚至是九分的罕见啊（悲</del>

**Comment by hejiehao** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/d5dbf901-5327-4c9a-9af4-cab36a731b4f)

**Reply by iPodtouch6557** [reply]
我是100:)

**Reply by MoYuan-CN** [reply]
Release = 1000
SnapShot = 100
看起来是更新器抽风了 见 https://github.com/Hex-Dragon/PCL2/commit/f0c634db7802719631083a486b23118cc0fd22cf

**Reply by kjjjjjjjjjjjj** [reply]
Well we set your account and your problem is solved

**Reply by MoYuan-CN** [reply]
right. this version code problem was resolved (view #2813)
